### PR TITLE
Button Text Color Accessibility Fix - Update on Input Focus Change

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* Update text color on buttons when focus of input fields changes

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java [hunk 1]
```diff
@@ -135,6 +137,13 @@ public void afterTextChanged(final Editable editable) {
                 setJoinButtonState();
             }
         });
+        setupName.setOnFocusChangeListener((view, hasFocus) -> {
```
Comment (waynemo) [inline AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java:140]: Could this be done with a selector?

input_text_selector.xml
```
<selector xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:state_focused="true" android:color="@color/colorAccent" />
    <item android:color="@color/textbox_secondary" /> <!-- default case -->
</selector>
```

TextView property:
```
<TextView
    android:textColor="@‌​drawable/input_text_selector" />
```

**Review by waynemo — APPROVED** [review]

**Review by waynemo — CHANGES_REQUESTED** [review]
See comments above

File: AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java (hunk 1 above)
Comment (iaulakh) [reply AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java:140]: I tried this approach before. It does not work as we do not set any focus on TextView.

Comment (waynemo) [reply AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/SetupActivity.java:140]: I misunderstood the change. I updated the PR title and description to better match the changes.

**Review by waynemo — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Pull request merged**